__tests__/app.test.js

```diff
@@ -116,3 +118,107 @@ describe(`GET /api`, () => {
+                    review_body: 'Fiddly fun for all the family',
+                    category: 'dexterity',
+                    created_at: "2021-01-18T10:01:41.251Z",
+                    votes: 2,
+                };
+                return request(app)
+                    .patch(`/api/reviews/2`)
+                    .send(updateVote)
+                    .expect(202)
+                    .then(({ body }) => {
+                        const { review } = body;
+                        expect(review).toEqual(updatedReview);
+                    });
+            });
+            test('ERROR non-existent review id returns 404 not found', () => {
+                const updateVote = { inc_votes: 7 };
+                return request(app)
+                    .patch('/api/reviews/999999')
+                    .send(updateVote)
+                    .expect(404)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('review_id not found');
+                    });
+            });
+            test('ERROR invalid review id returns 400 bad request', () => {
+                const updateVote = { inc_votes: 7 };
+                return request(app)
+                    .patch('/api/reviews/nine')
+                    .send(updateVote)
+                    .expect(400)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('bad request - review_id is not a number');
+                    });
+            });
+            test('ERROR invalid inc_votes returns 400 bad request', () => {
+                const updateVote = { inc_votes: 'seven' };
+                return request(app)
+                    .patch('/api/reviews/2')
+                    .send(updateVote)
+                    .expect(400)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('bad request - inc_votes is not a number');
+                    });
+            });
+            test('ERROR votes should never be negative returns 400 bad request', () => {
+                const updateVote = { inc_votes: -99 };
+                return request(app)
+                    .patch('/api/reviews/2')
+                    .send(updateVote)
+                    .expect(400)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('bad request - total votes cannot be negative');
+                    });
+            });
+            test('ERROR inc_votes has not been sent', () => {
+                const updateVote = {};
+                return request(app)
+                    .patch('/api/reviews/2')
+                    .send(updateVote)
+                    .expect(400)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('bad request - inc_votes has not been sent');
+                    });
+            });
+        })
+        
+    });
```

Comment:
Really good test suite here. Semantics, formatting and error scenarios are all top notch.